README.md

```diff
@@ -128,25 +116,6 @@ $$
     FROM pg_sequences WHERE schemaname = schema_name AND sequencename = sequence_name;
 $$ LANGUAGE sql VOLATILE SECURITY DEFINER;
 
---- For Postgres 9.6 and older, use this:
```

Comment:
There is "--- The following is needed for Postgres 10+:" above that we can remove.

Reply:
Eh, thinking about this, we presumably still need that (on all versions now) if you were to run the sequence report. Which we no longer really do. I think the README needs some TLC anyway (our docs on [pganalyze.com](https://pganalyze.com) are much more up-to-date), so I think I'll just leave as-is for now.